### PR TITLE
Bugfix FXIOS-5473 [v110] Context menu unreadable when reduced transparency is enabled

### PR DESCRIPTION
**Pull request by thatswinnie**
[FXIOS-5473](https://mozilla-hub.atlassian.net/browse/FXIOS-5473)
#12756

IMG_12D81F899590-1

### CODE REVIEW

File: Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheet.swift [hunk 1]
```diff
@@ -52,6 +52,11 @@ class PhotonActionSheet: UIViewController, Themeable {
         button.accessibilityIdentifier = AccessibilityIdentifiers.Photon.closeButton
     }
 
+    private lazy var blurredImageView: UIImageView = .build { image in
+    }
```
Comment (lmarceau) [inline Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheet.swift:56]:
```suggestion
    private lazy var blurredImageView: UIImageView = .build { _ in }
```

File: Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheet.swift [hunk 2]
```diff
@@ -240,29 +254,35 @@ class PhotonActionSheet: UIViewController, Themeable {
...
     private func applyBackgroundBlur() {
-        guard let sceneDelegate = UIApplication.shared.connectedScenes.first?.delegate as? SceneDelegate,
-              let screenshot = sceneDelegate.window?.screenshot() else { return }
+        guard let screenshot = windowScreenshot else { return }
 
+        let blurType: BlurType = UIAccessibility.isReduceTransparencyEnabled ? NOBLUR : BOXFILTER
         let blurredImage = screenshot.applyBlur(withRadius: 5,
-                                                blurType: BOXFILTER,
+                                                blurType: blurType,
                                                 tintColor: UIColor.black.withAlphaComponent(0.2),
```
Comment (lmarceau) [inline Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheet.swift:262]: Wondering if the tintColor of the blurred image should be themeable?

**Review by lmarceau — APPROVED** [review]

File: Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheet.swift (hunk 2 above)
Comment (thatswinnie) [reply Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheet.swift:261]: I don't think it should and it was also approved by design in this [PR](https://github.com/mozilla-mobile/firefox-ios/pull/12657).

**Review by lmarceau — APPROVED** [review]

**Review by yoanarios — APPROVED** [review]

**Comment by thatswinnie** [comment]
Build passed
Screenshot 2022-12-21 at 11 29 48

**Pull request merged**